python/cuml/dask/manifold/umap.py

```diff
@@ -95,7 +98,7 @@ def __init__(self, model, client=None, **kwargs):
         self._set_internal_model(model)
 
     def transform(self, X, convert_dtype=True):
-        """
+        r"""
```

Comment:
Good catch. I was seeing some weird errors with hyperlinks and tried to convert to a literal string to avoid any html link issues. It did not work so this can be removed.